Feedback

Sofilus opened the pull request.

Sofilus committed:
Signed contract SO

TessDevon committed:
Added my name

c-ridesjo committed:
ändrar bokstav

c-ridesjo committed:
Merge branch 'main' of https://github.com/Medieinstitutet/fed22d-js-grundkurs-gruppuppgift-webbshop-team-gummianka

Sofilus committed:
Added name

TessDevon committed:
Added donut images

c-ridesjo committed:
Merge pull request #5 from Medieinstitutet/images

Added donut images

TessDevon committed:
Added wireframe

TessDevon committed:
Merge pull request #6 from Medieinstitutet/wirefraime

Added wireframe

c-ridesjo committed:
add html

Sofilus committed:
Merge pull request #7 from Medieinstitutet/index.html

add html

postmodernistx reviewed (commented):
Hej! 👋 
Här kommer en första koll. Kanon att ni har kommit igång!

- [ ] Undvik å, ä och ö i filnamn (munkpåfat.jpg, nötmunk.jpg)

TessDevon committed:
Add html boilerplate, meta tags.
Linked in css and js.

c-ridesjo committed:
Merge pull request #8 from Medieinstitutet/Add-basic-structure

Add html boilerplate, meta tags.

postmodernistx reviewed (commented):
👍

Sofilus committed:
Added html and changed name img

Sofilus committed:
added br to form html and added scss files

TessDevon committed:
Merge pull request #9 from Medieinstitutet/Html

Added html and changed name img

TessDevon committed:
Delited files from HTML boilerplate

Sofilus committed:
Created dounat cards

TessDevon committed:
Styled css for mobile

TessDevon committed:
Added css mobile for basket and form.
Change tabletview in cscc with a few pixels to 768.

TessDevon committed:
Basketfixes

Sofilus committed:
Merge pull request #10 from Medieinstitutet/Add-CSS-in-mobile,-for-buket-and-form

Add css in mobile, for basket and form

Sofilus committed:
Created information box

postmodernistx reviewed (commented):
Undrar kring denna namngivning:
![image](https://user-images.githubusercontent.com/18628999/201413169-a08f992f-b647-4885-8300-13f976ed21ca.png)

Ska filen `_desctop.scss` heta `_desktop.scss` istället?

TessDevon committed:
Styled CSS in tablet and desktop

TessDevon committed:
Fixes in css. Cleane code in css.

TessDevon committed:
Fixes css button.

Sofilus committed:
Merge pull request #11 from Medieinstitutet/Add-CSS-to-plate-and-laptop,-basket-and-form

Add css to plate and laptop, basket and form

Sofilus committed:
Created and designed dropdown donuts

Sofilus committed:
Created header to our donuts part

Sofilus committed:
Added css tablet

TessDevon committed:
Fixed form-input

TessDevon committed:
Cleaned code in css

Sofilus committed:
Merge pull request #12 from Medieinstitutet/FixesForm-CSS

Fixes form css

Sofilus committed:
Created desktop size dounat and information

c-ridesjo committed:
Add header, nav and footer

c-ridesjo committed:
Deleted images

TessDevon committed:
Merge branch 'main' into dounat-cards

TessDevon committed:
Merge pull request #13 from Medieinstitutet/dounat-cards

Dounat cards

TessDevon committed:
Merge branch 'main' into Header,-nav,-footer

TessDevon committed:
Merge pull request #14 from Medieinstitutet/Header,-nav,-footer

Header, nav, footer

c-ridesjo committed:
fixes, css, html

c-ridesjo committed:
fixed css

c-ridesjo committed:
add js_donuts_button

Sofilus committed:
Merge pull request #15 from Medieinstitutet/css-fix,-js-knappar

Css fix, js knappar

TessDevon committed:
Added pattern in HTML forms

c-ridesjo committed:
Merge pull request #16 from Medieinstitutet/Added-parts-in-HTML--Form

Added pattern in HTML forms

postmodernistx reviewed (commented):
Kör på 🍁

TessDevon committed:
Added Prettier and ESLint

Sofilus committed:
Merge pull request #17 from Medieinstitutet/Add-Prettier-and-ESLint

Added Prettier and ESLint

TessDevon committed:
Added JS to all donutbuttons, don't work. Ask Jenni tomorrow.

TessDevon committed:
Added  JS to form

TessDevon committed:
Added JS to form

Sofilus committed:
Merge pull request #18 from Medieinstitutet/Add-JS-to-Donutbotton

Add js to donutbotton

Sofilus committed:
JS Add and subtrace buttons, ssn and cardinfo

TessDevon committed:
Merge pull request #19 from Medieinstitutet/jsCardsAndForm

JS Add and subtrace buttons, ssn and cardinfo

TessDevon committed:
Fixed Prittier

TessDevon committed:
Fixes image

Sofilus committed:
Merge pull request #20 from Medieinstitutet/Fix-Prittier

Fix prittier

c-ridesjo committed:
Fixes html och css

Sofilus committed:
Placed donuts in array and object

TessDevon committed:
Merge pull request #21 from Medieinstitutet/Change-donuts-to-arrays-and-objekts

Placed donuts in array and object

postmodernistx reviewed (commented):
Bra, kör på!

Sofilus committed:
amount change when + and - is clicked

TessDevon committed:
Added margin to form

TessDevon committed:
Fixes css form

TessDevon committed:
added placeholder in html

Sofilus committed:
Created donuts to basket in html and css

TessDevon committed:
Added JS to Form.
 Disabled Button until everithing is correct. Delited Pattern and moved Regexp to JS.

Sofilus committed:
Merge pull request #22 from Medieinstitutet/TessTestForm

Tess test form

Sofilus committed:
Choosen donuts appear in basket with amount

Sofilus committed:
Donut price in basket

TessDevon committed:
Fixes FormSubmitButton:
Disable again if something is false after everything is true. Moved checkbox for GDPR so it's the last part of the form so the user don't send the form when he/she push enter to times after the last inputfeild.

c-ridesjo committed:
Merge pull request #24 from Medieinstitutet/FixesJSForm

Fixes FormSubmitButton:

TessDevon committed:
Merge branch 'main' into Jennifixes

TessDevon committed:
Merge pull request #23 from Medieinstitutet/Jennifixes

Fixes html och css

Sofilus committed:
Totalsum updates price

Sofilus committed:
Made totalsum able to go to 0 SEK

Sofilus committed:
Added comments

TessDevon committed:
Added swapImages to donuts in JS

TessDevon committed:
Added SwapImages to Donut in JS

TessDevon committed:
Merge pull request #26 from Medieinstitutet/AddSlideShowOnDonutcard

Add slide show on donutcard

TessDevon committed:
Merge branch 'main' into jsSaveAmountValue

TessDevon committed:
Merge pull request #25 from Medieinstitutet/jsSaveAmountValue

Js save amount value

TessDevon committed:
Added comments to the Javascript
 I wrote last braunch.

TessDevon committed:
Fixes

TessDevon committed:
Added Luciadonut

TessDevon committed:
Fixes

TessDevon committed:
Added Christmasteams

Sofilus committed:
Merge pull request #27 from Medieinstitutet/SpecialrulesInvoiceDiscount0,1

Specialrules invoice discount0,1

Sofilus committed:
Made it possible to type in amount in donutcards

Sofilus committed:
Added empty basket btn and made it work

Sofilus committed:
Created discount input and made it work

Sofilus committed:
Shoppingcart icon shows total price

TessDevon committed:
Merge pull request #28 from Medieinstitutet/BaketAndBasketIcon

Baket and basket icon

TessDevon committed:
Fixes problem last marge so we can look at the code after lunch.

Sofilus committed:
Changed typed in amount from string to number

Sofilus committed:
changed place discountfield and btn

Sofilus committed:
If the discountcode is wrong totalprice returns

Sofilus committed:
If 10 >= donuts ordered 10% discount on that donut

Sofilus committed:
created shipping hmtl

TessDevon committed:
Cleaned coment don't needed in Js,
 or moved them so it's easyer to fiind in the bigger sections.

Sofilus committed:
Partprice updates when discont 10%

TessDevon committed:
Same like the one before

TessDevon committed:
Updated SantaSection with green color.

TessDevon committed:
updates santasection

TessDevon committed:
Changes in CSS Form

TessDevon committed:
Added no Invoice over 800 kr in Form.
Change to Summary when it's added later.

TessDevon committed:
Added JS comments in "No invoice over 800".

TessDevon committed:
Added comment iin christmaspart.

postmodernistx reviewed (commented):
👍 Feedback kommer här!

TessDevon committed:
ClearFormTimer with all parts are done.
Started to do a PopupArea to the resetbutton.
Style to tablet and desktop are left. Also JS.

TessDevon committed:
Styles popuparea, JS done.
Problem. Asked J for help.

TessDevon committed:
Fixes

TessDevon committed:
Cleaned and corrected in CSS.

TessDevon committed:
Fixes, styles and test of popupArea.

TessDevon committed:
Form backgrundimage-size cover.
So the image dont repeat itself.

Sofilus committed:
Shipping price change auto, price updates fri-sun

Sofilus committed:
monday before 10, 10% discount on total price

Sofilus committed:
Change structure and added tuesday discount

Sofilus committed:
Added price to pay

TessDevon committed:
Merge branch 'main' into smallFixesBasketAndSpecialRulesDiscount

TessDevon committed:
Merge pull request #29 from Medieinstitutet/smallFixesBasketAndSpecialRulesDiscount

Small fixes basket and special rules discount

TessDevon committed:
Added today2

TessDevon committed:
Merge branch 'main' into AddJsToDeliveryTimeTess

Sofilus committed:
Merge pull request #30 from Medieinstitutet/AddJsToDeliveryTimeTess

Added Deliverytime, popup, fixes

TessDevon committed:
Fixes

TessDevon committed:
Fixes in popup.
Added toFixed to sum and startShippingSum. Added Donutamout and totalprice to popup.
Fixed over 800 kr to the right value.

Sofilus committed:
Merge pull request #31 from Medieinstitutet/Popupvalue-and-css

Fixes in popup.

TessDevon committed:
Added more coments in the part i coded.

Sofilus committed:
added base to sorting

Sofilus committed:
Created sorting by price

Sofilus committed:
Made it possible to choose pricerange

Sofilus committed:
Made buttons klickable

TessDevon committed:
Added READMEmap, map for validation and printscreens and added text in humans.

TessDevon committed:
Fixes REDAME and assets

TessDevon committed:
Added CSS to Santateam

TessDevon committed:
Added images to Printscreenmap

TessDevon committed:
Added images in README. As files.

Sofilus committed:
Fixed basket price with cupong code

Sofilus committed:
Merge pull request #32 from Medieinstitutet/Fixes-coment

Added more coments in the part i coded.

TessDevon committed:
Merge branch 'main' into sortDonuts

TessDevon committed:
Merge pull request #33 from Medieinstitutet/sortDonuts

Sort donuts - All sorting options fully functional

TessDevon committed:
Added image of popup in README

c-ridesjo committed:
css fix header and footer

TessDevon committed:
Merge branch 'main' into css-fix-header-and-footer

TessDevon committed:
Merge pull request #35 from Medieinstitutet/css-fix-header-and-footer

css fix header and footer

TessDevon committed:
Added parst in Readme.
Techstach, Git live, links to pictures.

TessDevon committed:
Added link to picture i readme

TessDevon committed:
Fixes Readme

Sofilus committed:
Merge pull request #36 from Medieinstitutet/CSSsheckSanta

Cs ssheck santa

postmodernistx reviewed (commented):
OBS! Allt jag _inte_ har kommenterat är BRA!!!

Sofilus committed:
Price range goes between 15-30

Sofilus committed:
Added width and height on img html

Sofilus committed:
Wrote new alt text and added it to img

Sofilus committed:
some css basket and added empty basket js

Sofilus committed:
basket css base

Sofilus committed:
css basket base

Sofilus committed:
css basket tablet

Sofilus committed:
css basket desktop

c-ridesjo committed:
fix css base and tablet

postmodernistx committed:
:bug: bugfix

Vet inte varför typeAmountInput hela tiden gav att den inte är defined - kunde inte hitta orsaken till det.

TessDevon committed:
Fixes CSS Category

c-ridesjo committed:
css_tries

Sofilus committed:
small css change basket

Sofilus committed:
small changes css

Sofilus committed:
small css changes

Sofilus committed:
added and fixed meta tags

Sofilus committed:
added z-index on basket icon

c-ridesjo committed:
changed text color h1

Sofilus committed:
Updated comments and added missing semicolon

Sofilus committed:
moved a out of ul

TessDevon committed:
Fixxes CSS

c-ridesjo committed:
tiny fixes

Sofilus committed:
Merge pull request #38 from Medieinstitutet/LastFixesCSSandJSTess

Fixxes CSS

TessDevon committed:
Merge branch 'main' into Correction-css-header,-lighthouse

TessDevon committed:
Merge pull request #39 from Medieinstitutet/Correction-css-header,-lighthouse

Correction css header, lighthouse

TessDevon committed:
Merge branch 'main' into JsPricerangeCssAndSmallChanges

TessDevon committed:
Merge pull request #37 from Medieinstitutet/JsPricerangeCssAndSmallChanges

Js pricerange css and small changes

Sofilus committed:
sorting area desktop css

Sofilus committed:
moved pink value css sorting area

Sofilus committed:
changed margin tablet pink number sorting area

Sofilus committed:
css changes

TessDevon committed:
Merge branch 'main' into css-fix-sort/filter

TessDevon committed:
Merge pull request #40 from Medieinstitutet/css-fix-sort/filter

Css fix sort/filter

TessDevon committed:
Normalize

TessDevon committed:
Revert "Normalize"

This reverts commit 447b4a957ceb1cc1a3f43b005b8e96b4423fcc3e.

TessDevon committed:
Revert "Merge pull request #40 from Medieinstitutet/css-fix-sort/filter"

This reverts commit 494cee923e561136fa05d79d565d90681cffd74e, reversing
changes made to 24108880534753f8860d55e4af0e3e2ba5089ac1.

postmodernistx committed:
Update _desktop.scss

postmodernistx committed:
Merge branch 'main' into revert-stuff

* main:
  Revert "Merge pull request #40 from Medieinstitutet/css-fix-sort/filter"
  Revert "Normalize"

# Conflicts:
#	style/_desktop.scss

postmodernistx committed:
:lipstick: CSS fixes for category filters

TessDevon committed:
CSSfixesInform

TessDevon committed:
Added Style to sorting.
Fixed Luciadonut so it added after the first donut.
Must order at least one donut to make an order.

TessDevon committed:
Fixed codeindentation in JS

Sofilus committed:
Merge pull request #41 from Medieinstitutet/FixesLuciadonut

Fixes luciadonutbug, added CSS in sortpart all three sizes, intendation in JS.

c-ridesjo committed:
spell check website

c-ridesjo committed:
spell check comments

TessDevon committed:
Merge pull request #42 from Medieinstitutet/Validation-html/css-etc

Validation html/css etc

Sofilus committed:
fixt bugg with partsum in basket

Sofilus committed:
FIxed bugg, discountcode disapear when emptybasket

Sofilus committed:
Bugg fixed, part sum rond off to integer

Sofilus committed:
bf, empty basket disapear if donuts added

TessDevon committed:
Bugfix.
Submitbutton updates when basket is updated.

Sofilus committed:
bugg fix monday befor 10 10%

Sofilus committed:
bf, made basket icon not wisable when popup

Sofilus committed:
removed icon information part

TessDevon committed:
Added pacereload om popupbutton

Sofilus committed:
removed date

Sofilus committed:
removed br monday discount

TessDevon committed:
Fixes subitbutton lagerscreen

Sofilus committed:
Merge pull request #44 from Medieinstitutet/FinalBugsTess

Final bugfixes tess

TessDevon committed:
Merge branch 'main' into JsDebugg

TessDevon committed:
Merge pull request #43 from Medieinstitutet/JsDebugg

Js debugg

TessDevon committed:
Noralize

TessDevon committed:
Added Images of our site. in README

c-ridesjo committed:
validation html, css

TessDevon committed:
Added Santatheme

Sofilus committed:
Merge pull request #46 from Medieinstitutet/AddImages

Add images

TessDevon committed:
Fixes Santa

TessDevon committed:
Merge pull request #45 from Medieinstitutet/validate

validation html, css

c-ridesjo committed:
add validationsreports

TessDevon committed:
Merge pull request #47 from Medieinstitutet/add-validation-images

add validationsreports

TessDevon committed:
FixesLighthouse

Sofilus committed:
Merge pull request #48 from Medieinstitutet/lighthouseFixes

FixesLighthouse

c-ridesjo committed:
last check

TessDevon committed:
Merge pull request #49 from Medieinstitutet/check-availibility

last check
